.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: Test suite
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.7]
+
+    steps:
+    - name: Checkout repo
+      uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install software
+      run: |
+        pip install .[all]
+    - name: Test
+      run: |
+        pytest -n 2 -vv --doctest-modules --cov dmriprep --cov-config .coveragerc --cov-report xml:cov.xml dmriprep
```

Comment:
the options to pytest would be better set within the setup.cfg file.

Reply:
Done in 0cc7787